### DISCUSSION TITLE
Sorting of articles within a tag

### DISCUSSION THREAD

**Discussion by snjallingur** [squidfunk/mkdocs-material#7096] [Ideas]
Hi,
I am using the insider tag plugin to organize articles. E.g. I have a separate page for each tag and those pages only display articles for this tag. This is working fine. However I would like to find a way to affect the ordering of the articles on the page. I have a page that shows me release notes. This site is displaying the release notes in a ascending order. E.g. 4.8, 4.9 and so on. But obviously I want to list the articles with the newest first. So it should display 4.9. before 4.8.
This is the code I am using to only display articles with the tag "Releases":
`<!-- material/tags { include: [Releases] } -->`
Is that achievable?

Another related question to this: can I influence how the page is displayed (the UI of the page?)
br

**Comment by alexvoss** [comment]
I assume you want this only for the releases and not for other tag listings? AFAIK there is only the global option https://squidfunk.github.io/mkdocs-material/plugins/tags/#config.listings_sort_reverse at the moment. Seems like a sensible thing to want to override per listing. Some options can already be overridden but the sorting options do not seem to be part of that group.

**Reply by squidfunk** [reply]
Jup, definitely a sensible feature request.
